readme.md

```diff
@@ -77,3 +77,45 @@ make call to `http://localhost:8999/jobs/create?job_type=add&job_parameters={%22
 ## Extending
 New jobs can be added by adding a new Celery function to the jobs folder and adding 
 the function to the jobs registry. See the ./packer/jobs/example.py to learn how.
+
+
+## Existing jobs
+
+### Patient, diagnosis, biosource and biomaterial export
+
+Exports patient > diagnosis > biosource > biomaterial hierarchy as first 4 columns of the file.
+The rest of the columns are concepts. Higher level concepts (e.g Age that is specific to Patient level)
+get distributed to all rows specific to lower levels (e.g. Diagnosis)
+
+See [the CSR](https://github.com/thehyve/transmart-core/tree/dev/transmart-data/test_studies/CSR) test study as an example.
+
+#### Complete request body example
+
+```json
+{
+	"job_type":"patient_diagnosis_biosource_biomaterial_export",
+	"job_parameters": {
+		"constraint": {
+			"type":"study_name",
+			"studyId":"CSR"
+
+		},
+		"custom_name":"name of the export",
+		"row_filter": {
+			"type":"patient_set",
+			"subjectIds": ["P2", "P6"]
+		}
+	}
+}
+```
+
+where:
+
+ - `job_parameters.constraint` - any [transmart v2 api constraint](https://github.com/thehyve/transmart-core/blob/dev/open-api/swagger.yaml)
+or composition of them that used to get data from transmart.
+
+ - `custom_name` - name of the export job and the output `tsv` file.
```

Comment:
looks like it's also optional? 
and really small thing, maybe to be consistent in the description change `custom_name` to `job_parameters.custom_name`? Or `job_parameters.constraint` to `constraint`